examples/incredible-squaring/Makefile

```
@@ -0,0 +1,36 @@
+PHONY: deploy-el-and-avs-contracts
+PHONY: reset-anvil
+
+deploy-avs:
+	./contracts/anvil/deploy-avs.sh
+
+deploy-eigenlayer:
+	./contracts/anvil/deploy-eigenlayer.sh
+
+deploy-uam-permissions:
+	./contracts/anvil/uam-permissions.sh
+
+deploy-el-and-avs-contracts: deploy-eigenlayer deploy-avs deploy-uam-permissions
```

We should also add a rule to run `create-quorum.sh`

And:
```suggestion
deploy-el-and-avs-contracts: deploy-eigenlayer deploy-avs deploy-uam-permissions create-quorum
```

---

You can do something like [this](https://github.com/Layr-Labs/eigensdk-rs/blob/7884ed72e78f381a4ee437df30928897db6afda2/examples/awesome-vault-service/Makefile#L13-L16), so there’s no need to create the quorum in the example.

---

Done!